build.assets/tooling/cmd/release-notes/release_notes_test.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	_ "embed"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
```

Comment:
This could do with some additional tests to test the failure conditions as well.